docker/Dockerfile.deps

```diff
@@ -119,5 +119,14 @@ RUN FFMPEG_VERSION=4.2.1 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" && make install && \
     cd /tmp && rm -rf ffmpeg-$FFMPEG_VERSION
 
+# libsnd
+RUN LIBSND_VERSION=1.0.28                                                              && \
+    cd /tmp && wget http://www.mega-nerd.com/libsndfile/files/libsndfile-1.0.28.tar.gz && \
```

Comment:
We will host this binary as it is on LGPL - so you need to change it before merging to a different path.

Reply:
I followed precisely the same routine, which is a bit above - for `ffmpeg`, also on LGPL. What exactly should I change here?

Reply:
We are hosting the precise version of FFmpeg used `https://developer.download.nvidia.com/compute/redist/nvidia-dali/ffmpeg....`. We need to do the same for libsnd. So no `http://www.mega-nerd.com...`

Reply:
Will do